--- a/src/backend/mpv/CMakeLists.txt
+++ b/src/backend/mpv/CMakeLists.txt
@@ -10,7 +10,7 @@ set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTORCC ON) 
 set(CMAKE_AUTOUIC ON) 
 
-add_library(${PROJECT_NAME}
+add_library(${PROJECT_NAME} STATIC
 	mpvbackend.cpp  
 	mpvbackend.h  
 	qthelper.hpp
--- a/src/backend/scene/CMakeLists.txt
+++ b/src/backend/scene/CMakeLists.txt
@@ -12,7 +12,7 @@ set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTORCC ON) 
 set(CMAKE_AUTOUIC ON)
 
-add_library(${PROJECT_NAME}
+add_library(${PROJECT_NAME} STATIC
 	src/SceneViewer.cpp
 	src/SceneViewer.h
 )
--- a/src/backend/scene/lib/glad/CMakeLists.txt
+++ b/src/backend/scene/lib/glad/CMakeLists.txt
@@ -1,6 +1,6 @@
 project(glad)
 
-add_library(${PROJECT_NAME} include/glad/glad.h src/glad.c)
+add_library(${PROJECT_NAME} STATIC include/glad/glad.h src/glad.c)
 set_property(TARGET ${PROJECT_NAME} PROPERTY POSITION_INDEPENDENT_CODE ON)
 target_link_libraries(${PROJECT_NAME} PUBLIC ${CMAKE_DL_LIBS})
 target_include_directories(${PROJECT_NAME} PUBLIC include)
--- a/src/backend/scene/src/wallpaper/CMakeLists.txt
+++ b/src/backend/scene/src/wallpaper/CMakeLists.txt
@@ -9,7 +9,7 @@ if (CMAKE_BUILD_TYPE STREQUAL "Debug")
 	add_compile_definitions(DEBUG_OPENGL=1)
 endif ()
 
-add_library(${PROJECT_NAME}
+add_library(${PROJECT_NAME} STATIC
 	wallpaper.cpp
 	Image.cpp
 	WPJson.cpp
